components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -280,9 +280,9 @@ tweaking their $case/namelist_scream.xml file.
       <Casename>./${CASE}.scream</Casename>
       <Output__Control>
         <Frequency>${REST_N}</Frequency>
+        <Frequency__Units>${REST_OPTION}</Frequency__Units>
         <Frequency REST_OPTION="none">0</Frequency>
-        <Frequency__Units>INVALID</Frequency__Units>
-        <Frequency__Units REST_OPTION="nstep*">Steps</Frequency__Units>
+        <Frequency__Units REST_OPTION="none">INVALID</Frequency__Units>
```

Comment:
Perhaps we should remove this line. If `REST_OPTION=none`, wouldn't the output manager figure out that no output is required? I assume it would. And if not, we could easily add a few lines to the class.

Seems silly to add this extra line just to convert an arbitrary string "none" to another arbitrary string "INVALID" (and the latter is, I think, never used anyways).